turtl_backup/gui.py

```diff
@@ -2,25 +2,26 @@
 """
 
 import os
-from datetime import date
 import tkinter as tk
 import tkinter.filedialog as tkFileDialog
 import tkinter.messagebox as tkMessageBox
+from datetime import date
 from urllib.parse import urljoin
 
 import requests
-from turtl_backup import turtl_backup
+
+import turtl_backup
```

Comment:
I think you broke the gui here, there is probably no `build_basic_auth` in the turtl_backup package, but maybe in the module?

Reply:
fix in 1b9ac241ccf78e5e324c3157967b7ca407b694fa